_schedule/talks/2022-10-17-18-45-welcome-reception.md

```diff
@@ -0,0 +1,16 @@
+---
+accepted: true
+category: break
+date: 2022-10-17 18:45:00-07:00
+end_date: 2022-10-17 20:00:00-07:00
```

Comment:
We should probably have the start time at 5:45 or 6:00 since it'll be right after talks are done.

Reply:
@logankilpatrick I've moved it to 5:40 to match the online talk for better readability.
<img width="1250" alt="image" src="https://user-images.githubusercontent.com/7773256/195456061-d77bb98d-95b4-4143-9271-de3f77bf8cc4.png">

I also added a permalink to make it clickable:
<img width="1289" alt="image" src="https://user-images.githubusercontent.com/7773256/195456016-fc48d849-22c6-4be5-9571-16cdcf0093d4.png">

I also put in a horrible, dirty hack to fix #139. It's ugly but it works.  CC @jefftriplett for feedback on [4096936](https://github.com/djangocon/2022.djangocon.us/pull/216/commits/409693664ec5d582e3421714a8b2bdd383ac4dd9)